jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/ServiceHandlerTest.java

```diff
@@ -123,102 +133,76 @@ public void getServicesBothPortAndHealessTest() {
-        assertEquals(8080,services.get(0).getSpec().getPorts().get(0).getPort().intValue());
-        assertEquals("NodePort",services.get(0).getSpec().getType());
-        assertEquals("None",services.get(0).getSpec().getClusterIP());
-        assertTrue(services.get(0).getMetadata().getLabels().get("expose").equalsIgnoreCase("true"));
-        assertTrue(services.get(0).getMetadata().getName().equalsIgnoreCase("service-test"));
-    }
-
-    @Test
-    public void getServicesWithTCPProtocolTest() {
-
-        //checking protocol now
-        //TCP
-        port = ServiceConfig.Port.builder()
-                .port(8080).protocol("tcp").targetPort(80).nodePort(50).name("port-test").build();
-        ports.clear();
-        ports.add(port);
-        serviceConfigs.clear();
-        serviceconfig = ServiceConfig.builder()
-                .ports(ports).name("service-test").expose(true).headless(false).type("NodePort").build();
-
         //creating serviceConfigs of service;
         serviceConfigs.add(serviceconfig);
 
-        //calling the getServices Method
         services = serviceHandler.getServices(serviceConfigs);
-        assertEquals("TCP",services.get(0).getSpec().getPorts().get(0).getProtocol());
-    }
-
-    @Test
-    public void getServicesWithUDPProtocolTest() {
-        //UDP
-        port = ServiceConfig.Port.builder()
-                .port(8080).protocol("udp").targetPort(80).nodePort(50).name("port-test").build();
-        ports.clear();
-        ports.add(port);
-        serviceconfig = ServiceConfig.builder()
-                .ports(ports).name("service-test").expose(true).headless(false).type("NodePort").build();
-
-        //creating serviceConfigs of service;
-        serviceConfigs.clear();
-        serviceConfigs.add(serviceconfig);
 
-        //calling the getServices Method
-        services = serviceHandler.getServices(serviceConfigs);
-        assertEquals("UDP",services.get(0).getSpec().getPorts().get(0).getProtocol());
+        assertThat(services).first()
+            .satisfies(s -> assertThat(s.getSpec())
+                .hasFieldOrPropertyWithValue("type", "NodePort")
+                .hasFieldOrPropertyWithValue("clusterIP", "None")
+                .extracting(ServiceSpec::getPorts).asList()
+                .first()
+                .hasFieldOrPropertyWithValue("protocol", "TCP")
+                .hasFieldOrPropertyWithValue("name", "port-test")
+                .hasFieldOrPropertyWithValue("nodePort", 50)
+                .hasFieldOrPropertyWithValue("targetPort.intVal", 80)
+                .hasFieldOrPropertyWithValue("port", 8080))
+            .satisfies(s -> assertThat(s.getMetadata())
+                .hasFieldOrPropertyWithValue("name", "service-test")
+                .extracting(ObjectMeta::getLabels)
+                .asInstanceOf(InstanceOfAssertFactories.MAP)
+                .containsEntry("expose", "true"));
     }
 
     @Test
-    public void getServicesWithDefaultProtocolTest() {
```

Comment:
What happened to this test?

Reply:
@manusa whether we check for `null` protocol or `default`, we get `TCP` that's why this test has been parameterized into the `getServicesWithProtocol` test.

Reply:
And this test is checking `type` instead of `null` protocol https://github.com/eclipse/jkube/blob/c4000c845647f1c280e0680c3c9d3e391d90b9e7/jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/ServiceHandlerTest.java#L205-L223

So, should I remove it or rename it to `getServicesWithNullType` ?